### DISCUSSION TITLE
How do i change the site icon and title on the search engine

### DISCUSSION THREAD

**Discussion by melonmasteristaken** [community/community#106710] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

repo link:
github.com/1d10t1c-stud10s/more-to-explore

problem:
![IMG_1685](https://github.com/community/community/assets/112738649/94283aa7-4e9f-4693-a0f2-e12468463ed7)

Ive tried searching online for solutions but cant find anything that fixed my problem

**Comment by SaraRasoulian** [comment, marked as answer]
try meta tags:

```
<head>
  <meta charset="UTF-8">
  <meta name="description" content="More to Explore">
  <meta name="keywords" content="Explorer, Wiki, MtE">
  <meta name="author" content="John Doe">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
</head>
```

add favicon:
```
<head>
  <title>My Page Title</title>
  <link rel="icon" type="image/x-icon" href="/images/favicon.ico">
</head>
```

**Reply by SaraRasoulian** [reply]
try this issue: https://github.com/academicpages/academicpages.github.io/issues/301

**Reply by melonmasteristaken** [reply]
I added the meta tags but it still isn’t showing up, do i have to wait a bit? (It has already been deployed)

**Reply by SaraRasoulian** [reply]
I checked your website by inspect, meta tags must be inside head not body:

![image](https://github.com/community/community/assets/51083712/f6235299-45d6-410c-9df3-d168ce192048)
 
but now i think about it, i don't think meta tags could solve your issue, they are just for SEO purposes and nice to have

how about you try [custom domain](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/about-custom-domains-and-github-pages) for your pages?

**Reply by melonmasteristaken** [reply]
Is that the only way?

**Reply by chocmake** [reply]
Search engines periodically crawl sites for updates, so you won't see changes until they next crawl the site. You should see any new favicon and title on the Github Pages site itself though, if you added them correctly.

**Reply by melonmasteristaken** [reply]
Ah then it should work

**Reply by melonmasteristaken** [reply]
Thx

**Comment by rafidRD** [comment]
For the Icon Problem:
use [this](https://favicon.io/) for generating Icon on then search tab. The files are all provided download->unzip->upload

FOR THE NAME PRBLEM:
try this issue (if you are using jekyll): [Link](https://github.com/academicpages/academicpages.github.io/issues/301)

Click the accept button here👇🏼

**Reply by melonmasteristaken** [reply]
I already tried this one :/

**Reply by melonmasteristaken** [reply]
The favicon generator also didn’t work :(
